samples/MvcSample.Web/Models/Person.cs

```diff
@@ -0,0 +1,20 @@
+﻿// Copyright (c) Microsoft Open Technologies, Inc. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+
+namespace MvcSample.Web
+{
+    public class Person
+    {
+        [Required]
+        public string Name { get; set; }
+
+        [Required]
+        public Address Address { get; set; }
+
+        [Required]
+        public IEnumerable<Job> PastJobs { get; set; }
```

Comment:
expose this as a `List<Job>` rather than down-casting in `FormUrlEncodedController`.  as-is you're combining `as` with an immediate dereference -- an anti-pattern our samples should not demonstrate.